### DISCUSSION TITLE
Curiosity about a script.

### DISCUSSION THREAD

**Discussion by ARAHTHUL** [xiaoxiaoflood/firefox-scripts#125] [Requests]
Hello!

Today I found this page and I must thank you for all the work and support you've given.
There is something that I noticed in your screenshots and it's about a script named AddonsPage.uc.js that makes some changes that we can see in the last screenshot. My question is: may I have your permission to acces/download it, because it is not listed between the other scripts that can be downloaded? Or is something more personal?

Thank you in advance! Have a nice day!

**Comment by xiaoxiaoflood** [comment]
*AddonsPage.uc.js*, which wasn't written by me, creates the *userChrome JS* tab in *about:addons* next to *Extensions* and *Themes*. I haven't used it for years, so I don't have a working version.

You might want to try [this](https://u6.getuploader.com/script/download/2046), but I guess it won't work with my userChromeJS loader, it probably requires @alice0775's [version](https://github.com/alice0775/userChrome.js/blob/master/100/userChrome.js).

**Reply by ARAHTHUL** [reply]
Thank you for your answer! 

Indeed, I've tried @alice0775's version and it kinda works, as in I see the new tab but can not see the other changers, and that's because maybe I also have a .css in userChrome.css that I forgot. Anyway, it's not a big deal, as I've said it was more of a curiosity.

Keep up the good work! Have a nice one, sir!

**Reply by rediffusion** [reply]
@ARAHTHUL 
Can you show me the manual❓🐼
